metalware-new/src/commands/build.rb

```diff
@@ -113,7 +112,7 @@ def clear_up_built_node_marker_files
         FileUtils.rm_rf(files)
       end
 
-      def handle_interrupt
+      def handle_interrupt(_not_used)
```

Comment:
I was a bit confused by this parameter for a moment, though I now realise it's so the signature matches that in `BaseCommand`. Could be worth giving it the same name as in `BaseCommand` to make this clear? You can still prefix it with an underscore if you prefer to indicate it's required but intentionally not used.